README.md

```diff
@@ -1 +1,48 @@
-# listview-sort
\ No newline at end of file
+[![Build Status](https://travis-ci.org/mendixlabs/listview-filter.svg?branch=master)](https://travis-ci.org/mendixlabs/listview-filter)
+[![Dependency Status](https://david-dm.org/mendixlabs/listview-filter.svg)](https://david-dm.org/mendixlabs/listview-filter)
+[![Dev Dependency Status](https://david-dm.org/mendixlabs/listview-filter.svg#info=devDependencies)](https://david-dm.org/mendixlabs/listview-filter#info=devDependencies)
+[![codecov](https://codecov.io/gh/mendixlabs/listview-filter/branch/master/graph/listview-filter.svg)](https://codecov.io/gh/mendixlabs/listview-filter)
+![badge](https://img.shields.io/badge/mendix-7.6.0-green.svg)
+
+# Drop-down sort
+
+Add an interactive sort to your list view.
+It supports sorting on a single field similar to the built-in list view sort capabilities.
+
+## Features
+* Sort through a single field
+* Specify an option to sort list view items on load. If multiple defaults are selected, the widget will select the first one.
+* If no default select option is specified the widget will always select the first one.
+
+## Dependencies
+Mendix 7.6
+
+## Demo project
+
+[https://listviewsort.mxapps.io/](https://listviewsort.mxapps.io/)
+
+![Demo](/assets/demo.gif)
+
+## Usage
+
+### Data source configuration
+
+![Data source](/assets/Datasource.png)
+ - On the `List view entity` option of the `Data source` tab, browse and 
```

Comment:
Update to `List view with entity` and update screenshot

Reply:
Done